aws-iam-role-policies/variables.tf

```diff
@@ -0,0 +1,48 @@
+variable project {
```

Comment:
Please, add a variable for `iam_path` (and use it on all `iam` resources).

Reply:
added, and we can only use that for the IAM role in this module